### DISCUSSION TITLE
For static sites, map failed links in HTML files to the Markdown source files.

### DISCUSSION THREAD

**Discussion by jstirnaman** [lycheeverse/lychee#1469] [Q&A] [answered]
[Hyperlink provides an option](https://github.com/untitaker/hyperlink?tab=readme-ov-file#installation-and-usage) that tries to map links discovered in HTML files to their associated source files--for example, Markdown files in a static site. Can I do something similar using lychee?

**Comment by mre** [comment, marked as answer]
Not yet, but maybe someone wants to contribute it to lychee.
The code in question is here: https://github.com/untitaker/hyperlink/blob/bf165852f43428eb3476d8ed59b5406aafbdd31c/src/markdown.rs#L57-L69
It's also written in Rust by @untitaker.

lychee's parsing is not as sophisticated, but it should be doable to add. Don't hold your breath for it, though. 😆
